plenum/bls/bls_crypto_factory.py

```diff
@@ -28,7 +29,7 @@
 
 class BlsFactoryIndyCrypto(BlsFactoryCrypto):
     def __init__(self, basedir=None, node_name=None):
-        self._basedir = basedir
+        self._basedir = os.path.join(os.path.expanduser(basedir), "keys")
```

Comment:
I think this join needs to be on `BlsKeyManagerFile` level.

Reply:
moved to BlsKeyManagerFile